index.php

```diff
@@ -148,6 +148,10 @@
                     <img src="images/new-event.svg">
                     <span>Sign-Up for Event</span>
                 </div>
+                <div class="dashboard-item" data-link="editHours.php">
+                        <img src="images/add-person.svg">
+                        <span>See & Change My Event Hours</span>
```

Comment:
Maybe the wording here should be "View & Change My Hours", or we can call it the "Hour Manager"

Reply:
hour manager sounds confusing. view & change my hours is ok - though i may want to continue to say event hours to be more specific